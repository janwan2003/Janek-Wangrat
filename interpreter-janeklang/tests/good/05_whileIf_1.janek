int main() {
    int i = 0;
    while (i < 5) {
        print(i);
        i = i + 1;
    }
    if (i == 5) {
        print("i is 5");
    }
    return 0;
}
