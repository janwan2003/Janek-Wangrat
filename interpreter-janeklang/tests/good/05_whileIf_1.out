0
1
2
3
4
i is 5
